lib/common/common.go

```diff
@@ -78,6 +78,31 @@ func HexToBytes(in string) ([]byte, error) {
 	return out, err
 }
 
+// MustHexToBytes turns a 0x prefixed hex string into a byte slice
+// it panic if it cannot decode the string
+func MustHexToBytes(in string) []byte {
+	if len(in) < 2 {
```

Comment:
I like this, should we replace usages of `HexToBytes` with this?  Or is there use for `HexToBytes`?

Reply:
I think usages in the tests could be replaced, but in the actual code let's leave it as is, so it doesn't panic